install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1602,9 +1602,21 @@ localRedirectPolicy: false
 # To include or exclude matched resources from cilium identity evaluation
 # labels: ""
 
-# logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# -- (string) Allows you to define logging options.
+# LogOpt sets log driver options for cilium
+# "log-opt", "Log driver options for cilium-health, for example syslog.level=info,syslog.facility=local5,syslog.tag=cilium-agent"
+# level, network (syslog.LOG_KERN(3)), address, severity(loglevel), facility
+# eg: "level=warn,format=(text|json|json-ts),syslog.facility=local4,syslog.severity=info,syslog.tag=cilium"
```

Comment:
Sorry, I don't think this is very clear. Could you please turn this into a proper sentence?